README.md

```diff
@@ -0,0 +1,227 @@
+*   [OpenFermion-Dirac](https://github.com/bsenjean/Openfermion-Dirac) to support integration with
+    [DIRAC](http://diracprogram.org/doku.php).
+*   [OpenFermion-QChem](https://github.com/qchemsoftware/OpenFermion-QChem) to support integration
+    with [Q-Chem](https://www.q-chem.com).
+
+## Documentation
+
+Documentation for OpenFermion can be found at
+[quantumai.google/openfermion](https://quantumai.google/openfermion) and the following links:
+
+*   [Installation](https://quantumai.google/openfermion/install)
+*   [API Docs](https://quantumai.google/reference/python/openfermion/all_symbols)
+*   [Tutorials](https://quantumai.google/openfermion/tutorials/intro_to_openfermion)
+
+You can run OpenFermion's interactive Jupyter Notebooks, such as the introductory tutorial, in
+[Colab](https://colab.research.google.com/github/quantumlib/OpenFermion/blob/master/examples/intro_to_openfermion.ipynb)
+or
+[MyBinder](https://mybinder.org/v2/gh/quantumlib/OpenFermion/master?filepath=examples%2Fintro_to_openfermion.ipynb).
+
+## Contributing to OpenFermion
+
+We'd love to accept your contributions and patches to OpenFermion. There are a few small guidelines
+you need to follow.
+
+*   Contributions to OpenFermion must be accompanied by a Contributor License Agreement (CLA). You
+    (or your employer) retain the copyright to your contribution; the CLA simply gives us permission
+    to use and redistribute your contributions as part of the OpenFermion project. Please visit
+    https://cla.developers.google.com/ to see your current agreements on file or to sign a new one.
+*   All submissions, including submissions by project members, require review. We use GitHub pull
+    requests for this purpose. Consult the appropriate [GitHub Help
+    documentation](https://help.github.com/articles/about-pull-requests/) for more information on
+    using pull requests.
+*   Please make sure your new code comes with extensive tests! We use automatic testing to make sure
+    all pull requests pass tests and do not decrease overall test coverage by too much.
+*   Please also make sure to follow the OpenFermion source code style. We mostly follow Python's
+    [PEP 8](https://www.python.org/dev/peps/pep-0008/) guidelines and use the corresponding
+    [linter](https://pypi.python.org/pypi/pep8) to check for it.
+*   Code should always be accompanied by documentation. Formatted OpenFermion documentation is
+    generated automatically and can be found [on the Quantum AI web
+    site](https://quantumai.google/openfermion).
+*   We use [GitHub issues](https://github.com/quantumlib/OpenFermion/issues) for tracking requests
+    and bugs. Please post questions to the [Quantum Computing Stack
+    Exchange](https://quantumcomputing.stackexchange.com/) with an `openfermion` tag.
+
+## Citing OpenFermion<a name="how-to-cite-openfermion"></a><a name="how-to-cite"></a>
+
+When using OpenFermion for research projects, please cite:
```

Comment:
I think the previous version was more useful, because it had a link to the actual article.  The article page has a button to export citation in BibTeX or RIS format.

Can you replace the bibtex below with human readable citation (as was before) that has a link to the Journal article?  Feel free to shorten the authors list to *Jarrod R. McClean et al*.